golang/lib/networks/network_context.go

```diff
@@ -157,40 +157,33 @@ func (networkCtx *NetworkContext) RegisterFilesArtifacts(filesArtifactUrls map[s
 }
 
 // Docs available at https://docs.kurtosistech.com/kurtosis-client/lib-documentation
```

Comment:
Make sure to update the docs!

Reply:
Yep! I had the docs update in my burndown but not the typescript update, I'll add thanks!